Added notice about chai typings (#207)
openapi-library/OpenAPIValidators: packages/chai-openapi-response-validator/README.md

You don't need to `npm install --save-dev @types/chai-openapi-response-validator` because we [bundle our TypeScript Definition file into this package](https://github.com/openapi-library/OpenAPIValidators/blob/master/packages/chai-openapi-response-validator/index.d.ts).

> But make sure you have installed chai typings with `npm install --save-dev @types/chai` since chai does not provide typescript definition files.

Reviewer: ```suggestion
But make sure you `npm install --save-dev @types/chai` because Chai does not bundle their TypeScript definition files like this.
```
